src/cmake/testing.cmake

```diff
@@ -160,6 +160,11 @@ macro ( TESTSUITE )
             # and optimized
             add_one_testsuite ("${_testname}.optix.opt" "${_testsrcdir}"
                                ENV TESTSHADE_OPT=2 TESTSHADE_OPTIX=1 )
+            if (NOT EXISTS "${_testsrcdir}/NOFUSED")
```

Comment:
All the patch LGTM! I'm wondering about this /NOFUSED dir, where and how you generate it?

Reply:
I haven't added support for the new api to testrender yet, so the NOFUSED file is put in directories with optix tests that we want to skip testing with the new api. They're added by hand.